MyBOT/dist/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
     "author": {
-        "name": "mon#0010",
-        "id": "406184197405802497"
+        "name": "mon#0010 & Deivid#0045",

```

Comment:
You should put your name (without the #) and your profile id that you can find by turning on the Developer Mode on Discord

Reply:
alweady